### ISSUE TITLE
Rename `master` to `main`

### ISSUE THREAD

**Issue by SethTisue** [scala/scala.epfl.ch#173]
@fsalvi if I do this (following the instructions at https://github.com/github/renaming, would anything in the site publishing need to change?

or should I just go ahead, and we'll find out and deal with it?

**Comment by fsalvi** [comment]
Yes, you can go ahead and notify me. I think I only need to change the branch name in one place.

**Comment by SethTisue** [comment]
Great -- I've done all 3 repos, thanks!

**Comment by fsalvi** [comment]
Ok, I did the modifications.
